src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs

```diff
@@ -122,340 +105,391 @@ protected override void OnElementReady()
+            _bgImageObject.Show();
+            Children.Add(_bgImageObject);
+            _bgImageObject.Lower();
+        }
 
-            SetNativeView(_box);
+        void DeinitializeBgImageObject()
+        {
+            Children.Remove(_bgImageObject);
+            _bgImageObject.Unrealize();
+            _bgImageObject = null;
         }
 
-        void OnLayout()
+        void InitializeMoreOption()
         {
-            var rect = _box.Geometry;
-            Element.Layout(rect.ToDP());
-            _bgColorObject.Geometry = rect;
-            _bgImageObject.Geometry = rect;
+            _moreOption = new MoreOption(_mainLayout);
+            _moreOption.Show();
+            _moreOption.Clicked += OnMoreOptionItemClicked;
+            _moreOption.Closed += (s, e) => OnMoreOptionClosed();
+            _moreOption.Opened += (s, e) => OnMoreOptionOpened();
+            Children.Add(_moreOption);
+        }
 
-            _bgImageObject.StackAbove(_bgColorObject);
-            EvasObject prev = _bgImageObject;
+        void DeinitializeMoreOption()
+        {
+            Children.Remove(_moreOption);
+            _moreOption.Unrealize();
+            _moreOption = null;
+        }
 
-            IContainable<EvasObject> container = _box;
-            foreach (var obj in container.Children)
-            {
-                obj.StackAbove(prev);
-                prev = obj;
-            }
+        void UpdateBackgroundImage(bool initialize)
+        {
+            if (initialize && Element.BackgroundImageSource.IsNullOrEmpty())
+                return;
 
-            if (_actionButton != null)
+            if (Element.BackgroundImageSource is FileImageSource fileImageSource)
             {
-                var btnRect = _actionButton.Geometry;
-                var btnW = Math.Max(_actionButton.MinimumWidth, btnRect.Width);
-                var btnH = Math.Max(_actionButton.MinimumHeight, btnRect.Height);
-                var btnX = rect.X + (rect.Width - btnW) / 2;
-                var btnY = rect.Y + rect.Height - btnH;
-                _actionButton.Geometry = new Rect(btnX, btnY, btnW, btnH);
-                _actionButton.StackAbove(prev);
-                prev = _actionButton;
+                if (_bgImageObject == null)
+                {
+                    InitializeBgImageObject();
+                }
+                _bgImageObject.File = ResourcePath.GetPath(fileImageSource);
             }
-
-            _surfaceLayout.Geometry = rect;
-            _surfaceLayout.StackAbove(prev);
-            prev = _surfaceLayout;
-
-            if (_moreOption != null)
+            else
             {
-                _moreOption.Geometry = rect;
-                _moreOption.StackAbove(prev);
+                if (_bgImageObject != null)
+                {
+                    DeinitializeBgImageObject();
+                }
             }
         }
 
-        void UpdateBackground()
+        void UpdateActionButton(bool init)
```

Comment:
Use `init` to skip unnecessary code at initial time or just make `UpdateActionButton()` if you don't need.

Reply:
Done